kinder/pkg/cluster/manager/actions/kubeadm-init.go

```diff
@@ -101,8 +101,14 @@ func kubeadmInit(cp1 *status.Node, copyCertsMode CopyCertsMode, patchesDir, igno
 			// NB. certificate key is passed via the config file)
 		)
 	}
+
 	if patchesDir != "" {
-		initArgs = append(initArgs, "--experimental-patches", constants.PatchesDir)
+		if cp1.MustKubeadmVersion().LessThan(constants.V1_21) {
+			initArgs = append(initArgs, "--experimental-patches", constants.PatchesDir)
+		}
+		if err := copyPatchesToNode(cp1, patchesDir); err != nil {
```

Comment:
this is already being called here:
https://github.com/kubernetes/kubeadm/blob/ebd1ae0b04a22e94bc73d07f10a5c2416ac7afe0/kinder/pkg/cluster/manager/actions/kubeadm-init.go#L43-L51

i think we can remove the V1_19 constant and the code above.